coolwsd.xml.in

```diff
@@ -298,4 +298,8 @@
 
     <help_url desc="The Help root URL, or empty for no help (hides the Help buttons)" type="string" default="@HELP_URL@">@HELP_URL@</help_url>
 
+    <overwrite_mode>
+        <enable desc="Enable overwrite mode (user can use insert key)" type="bool" default="false">true</enable>
+    </overwrite_mode>
+
```

Comment:
Thanks, 

# Default value

As the CODE-only release has passed I would rather do the following:


- Update this PR make the default **true** (so COOL users that rely on overwrite mode do not suffer)
  - Then I will test it, and then merge
----

# Follow up PR
- Then another PR where we change the default:
Using the CODE flag to make it:
in **CODE**
#+begin_src xml
<overwrite_mode>
  <enable>false</enable>
</overwrite_mode>
#+end_src
and in **COOL**
#+begin_src xml
<overwrite_mode>
  <enable>true</enable>
</overwrite_mode>
#+end_src

----

# Future (possible) improvements
First time pressing the Insert triggers a snackbar

Sys admin in company A disabled feature. Everyone is happy but one guy.
The guy press Insert Key, sees the snackbar -> understands it and accepts it or goes annoying the sys admin
- If the feature is disabled we would show snackbar with timeout
  "Overwrite is disabled. Please contact admin."[Dismiss]

Sys admin in company B enables overwrite. Not everyone is happy but they at least can exit from overwrite mode to insert mode even if no insert key is available
- If the feature is enabled we would show snackbar with timeout
  "Overwrite mode is now enabled" [Disable] (Disable here means the same action as pressing again the **Insert** does)